### PR TITLE
Updated instruction for installing GPU driver on Linux*.

### PR DESCRIPTION
**Pull request by artanokhov**
### Tickets:
 - 79326

### CODE REVIEW

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 1]
```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Determine your operating system and it's version:
```
Comment (xu-yuan1) [inline docs/install_guides/configurations-for-intel-gpu.md:16]:
```suggestion
1. Check your operating system and its version: 
```

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 2]
```diff
@@ -13,31 +13,26 @@
...
-   not install a new driver. If the version of the driver is lower than the current version, the script uninstalls the lower version and installs the current version with your permission:
-   ![](../img/NEO_check_agreement.png) 
-
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
```
Comment (xu-yuan1) [inline docs/install_guides/configurations-for-intel-gpu.md:21]:
```suggestion
2. Install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your operating system. You can find instructions in the [Installation Guides of Intel® software packages for general purpose GPU capabilities](https://dgpu-docs.intel.com/installation-guides/index.html), or use one of the ways in our internal tests below:
```

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 3]
```diff
@@ -13,31 +13,26 @@
...
+--- |-------------------------
+Ubuntu 20.04 | [22.35.24055](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+Ubuntu 18.04 | [21.38.21026](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+CentOS 7 | [19.41.14441](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+RHEL 8 | [22.28.23726](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)                     
 
-3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
+> **NOTE**: To use the **Intel® Iris® Xe MAX Graphics**, see the [Intel® Iris® Xe MAX Graphics with Linux*](https://dgpu-docs.intel.com/devices/iris-xe-max-graphics/index.html) page for driver installation instructions.
+
+Additionally, it is recommended that you refer to the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/) for the latest instructions and recommendations on GPU driver installation.
```
Comment (xu-yuan1) [inline docs/install_guides/configurations-for-intel-gpu.md:32]: Here I feel a little confused. Step 2 has already provided two ways for installing OpenCL, and one of them provides the specific GitHub links for each OS.
This Intel® Graphics Compute Runtime Github page here says it provides the latest instructions and recommendation, which gives users too many choices and make it unclear on which one we are recommending.

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 2 above)
Comment (evgenytalanin-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:21]: @xu-yuan1 My English is not so good but this proposal does seem like correct re-wording. It's not about the latest driver vs other ways. It's about the latest version driver vs other versions of driver.

Comment (evgenytalanin-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:21]: > I think we can provide the link of the parent page to give some information on how to select the installation guides.

@xu-yuan1 My English is not so good but this proposal does seem like correct re-wording. It's not about the latest driver vs other ways. It's about the latest version driver vs other versions of driver.

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 3 above)
Comment (evgenytalanin-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:32]: > Here I feel a little confused. Step 2 has already provided two ways for installing OpenCL, and one of them provides the specific GitHub links for each OS. This Intel® Graphics Compute Runtime Github page here says it provides the latest instructions and recommendation, which gives users too many choices and make it unclear on which one we are recommending.

We do not recommend anything. It's the common misunderstanding that OpenVINO works only with recommended driver. Also we do not validate driver itself. We state which one was used in our internal validation, and this should not mean that others will not work or whatever. However, there might be different limitations for whatever version of driver, and users trying to install something on their own should be aware.

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 4]
```diff
@@ -13,31 +13,26 @@
...
```
Comment (evgenytalanin-intel) [inline docs/install_guides/configurations-for-intel-gpu.md:32]: This link (to Github) and the one in the beginning of step 2 (to dgpu-docs) are different but both are presented as something about the latest driver. Can we align this?

**Review by evgenytalanin-intel — CHANGES_REQUESTED** [review]

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 5]
```diff
@@ -13,31 +13,26 @@
...
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
+
+Operation System | Driver version
+--- |-------------------------
+Ubuntu 20.04 | [22.35.24055](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+Ubuntu 18.04 | [21.38.21026](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+CentOS 7 | [19.41.14441](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)  
+RHEL 8 | [22.28.23726](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)
```
Comment (ilya-lavrenov) [inline docs/install_guides/configurations-for-intel-gpu.md:28]: why all OSes are pointing to the same version?

Comment (ilya-lavrenov) [inline docs/install_guides/configurations-for-intel-gpu.md:28]: the link is the same for all Oses..

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 2 above)
Comment (xu-yuan1) [reply docs/install_guides/configurations-for-intel-gpu.md:21]: @evgenytalanin-intel , could you suggest which part of the sentence should be changed? this link: https://dgpu-docs.intel.com/index.html? or the last sentence "use one of the ways in our internal tests below"?
I was following the original sentence and didn't change much except the link. :)

**Review by p-durandin — CHANGES_REQUESTED** [review]

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 3 above)
Comment (xu-yuan1) [reply docs/install_guides/configurations-for-intel-gpu.md:32]: @evgenytalanin-intel Thanks, that helped clarify. We can reword the sentences to say "You can also select one from the following versions that we have tested: ..... 
To get the latest version and instructions, see the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/)."
Like you said, we do not recommend anything. Does it look good to you?

Comment (p-durandin) [reply docs/install_guides/configurations-for-intel-gpu.md:32]: To get the latest version and instructions, see the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/). for integrated graphics and https://dgpu-docs.intel.com/installation-guides/index.html for discrete ones"

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 6]
```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+1. Check your operating system and its version:
```
Comment (yury-gorbachev) [inline docs/install_guides/configurations-for-intel-gpu.md:16]: I do not think this step is necessary and if it is - it must be part of GPU driver manual probably.
Ideally we should just fall back to Gpu driver page and that's it.

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 7]
```diff
@@ -13,31 +13,26 @@
...
-   Higher hardware versions require a higher driver version, namely 20.35 instead of 19.41. If the script fails to uninstall the driver, uninstall it manually. During the script execution, you may see the following command line output:  
-   ```sh
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
+2. You can install the latest available **Intel® Graphics Compute Runtime for OpenCL™** for your OS, see the [Install Guides](https://dgpu-docs.intel.com/installation-guides/index.html) or choose one that we used in internal tests
+
+Operation System | Driver version
+--- |-------------------------
```
Comment (yury-gorbachev) [inline docs/install_guides/configurations-for-intel-gpu.md:24]: Do we have intention of updating this every time when we release something? I think this is not entirely sustainable process and perhaps we need to find some other way to tell which version we used during our validation. Now you are interrogating with installation instruction which can actually point to something else than GitHub repo.

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 8]
```diff
@@ -46,11 +41,6 @@ You've completed all required configuration steps to perform inference on proces
 
 ## Windows
 
-This section will help you check if you require driver installation. Install indicated version or higher.
-
-If your applications offload computation to **Intel® Integrated Graphics**, you must have the Intel Graphics Driver for Windows installed on your hardware.
-[Download and install the recommended version](https://downloadcenter.intel.com/download/30079/Intel-Graphics-Windows-10-DCH-Drivers). 
-
 To check if you have this driver installed:
```
Comment (yury-gorbachev) [inline docs/install_guides/configurations-for-intel-gpu.md:44]: I do not understand flow here. Instruction suggests to check which driver I have but what if I'm not? Where should I go?

**Review by yury-gorbachev — CHANGES_REQUESTED** [review]

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 4 above)
Comment (xu-yuan1) [reply docs/install_guides/configurations-for-intel-gpu.md:32]: > This link (to Github) and the one in the beginning of step 2 (to dgpu-docs) are different but both are presented as something about the latest driver. Can we align this?

What I meant was the Github link and the links in the table are from the same source. Now I see they are different in "tested versions" and "latest version". :)

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 9]
```diff
@@ -13,31 +13,26 @@
 
 If you have installed OpenVINO Runtime from the archive file, APT, or YUM, follow these steps to work with GPU:
```
[the remark this answers is not in the document]
Comment (xu-yuan1) [reply docs/install_guides/configurations-for-intel-gpu.md:14]: I assume this article is self-explaining as its heading, but yes we can make it more explicit by adding a leading paragraph. How about rewording a little bit as below: 
```
@sphinxdirective
.. _gpu guide:
@endsphinxdirective

To use OpenVINO™ GPU plugin and Intel® Processor Graphics (GPU) for inference, the Intel® GPU driver and Intel® Graphics Compute Runtime for OpenCL™ driver components must be properly configured on your system.

This article provides references to install those drivers on different systems.  You can also use the pre-installed driver on your system.

## Linux
```
Is that what you mean?

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 8 above)
Comment (helena-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:44]: Is it possible to have a Windows install without GPU drivers? GPU inference has always worked out of the box for me (on computers with an Intel iGPU). Could we have a sentence that GPU drivers are included in Windows and that it is not usually necessary to install anything, and then maybe add that if you want to use the validated version, do X?

Comment (evgenytalanin-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:44]: > Is it possible to have a Windows install without GPU drivers? GPU inference has always worked out of the box for me (on computers with an Intel iGPU). Could we have a sentence that GPU drivers are included in Windows and that it is not usually necessary to install anything, and then maybe add that if you want to use the validated version, do X?

I believe it works due to automatic updates which might not be the case everywhere. Also not sure how systems with iGPU and dGPU (of whatever vendor) are used and how updates should be handled there. So for me it worth mentioning how to install this.

**Review by p-durandin — APPROVED** [review]

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 10]
```diff
@@ -46,24 +30,31 @@ You've completed all required configuration steps to perform inference on proces
...
+## Additional info
+
+In the internal OpenVINO validation the following versions of Intel Graphics Driver were used:
+
+Operation System | Driver version
+--- |-------------------------
+Ubuntu 20.04 | [22.35.24055](https://github.com/intel/compute-runtime/releases/tag/22.35.24055)
+Ubuntu 18.04 | [21.38.21026](https://github.com/intel/compute-runtime/releases/tag/21.38.21026)
+CentOS 7 | [19.41.14441](https://github.com/intel/compute-runtime/releases/tag/19.41.14441)
+RHEL 8 | [22.28.23726](https://github.com/intel/compute-runtime/releases/tag/22.28.23726)
```
Comment (evgenytalanin-intel) [inline docs/install_guides/configurations-for-intel-gpu.md:56]: Since we list all the OSes here, we should add Windows as well. But this can be fixed later. Other than than LGTM

**Review by evgenytalanin-intel — APPROVED** [review]

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 11]
```diff
@@ -7,37 +7,21 @@
...
-   Add OpenCL user to video group    
-   ```
-   Ignore this suggestion and continue.<br>
-   You can also find the most recent version of the driver, installation procedure and other information on the [Intel® software for general purpose GPU capabilities](https://dgpu-docs.intel.com/index.html) site.
-
-3. **Optional:** Install header files to allow compilation of new code. You can find the header files at [Khronos OpenCL™ API Headers](https://github.com/KhronosGroup/OpenCL-Headers.git).
+Additionally, it is recommended that you refer to the [Intel® Graphics Compute Runtime Github page](https://github.com/intel/compute-runtime/) for the instructions and recommendations on GPU driver installation specific to particular release, including the list of supported hardware platforms.
 
 You've completed all required configuration steps to perform inference on processor graphics.
+
```
Comment (ilya-lavrenov) [inline docs/install_guides/configurations-for-intel-gpu.md:24]: question: if I follow all these instruction, will they recommend me to install only driver or OpenCL library as well?
Old script did both steps

**Review by kblaszczak-intel — COMMENTED** [review]
Minor rewording proposition for several paragraphs.

File: docs/install_guides/configurations-for-intel-gpu.md [hunk 12]
```diff
@@ -7,37 +7,21 @@
 
 @endsphinxdirective
 
+In case if you are intended to use OpenVINO GPU plugin and offload network inference to Intel® graphics processor, the Intel Graphics Driver should be properly configured on your system.
 
+> **NOTE**: In case you already have Intel Graphics Driver installed on your system, and you want to keep it, the installation steps are not required.
```
[the remark this answers is not in the document]
Comment (evgenytalanin-intel) [reply docs/install_guides/configurations-for-intel-gpu.md:12]: @kblaszczak-intel It's not about necessarily being up-to-date (in my view "up-to-date" means "the very latest"), it's about keeping already installed one. For various reasons (performance, not approved by security, some HW support is broken - there was such a case, etc.) users may prefer to use older versions.

File: docs/install_guides/configurations-for-intel-gpu.md (hunk 11 above)
Comment (artanokhov) [reply docs/install_guides/configurations-for-intel-gpu.md:23]: I've checked it on ubu20 and rhel8
### Ubuntu20
sudo -E apt-cache show intel-opencl-icd
Package: intel-opencl-icd
Provides: opencl-icd
Depends: libc6 (>= 2.29), libgcc-s1 (>= 3.4), libigdgmm11 (>= 20.1.1+ds1), libstdc++6 (>= 9), libigc1 (>= 1.0.3627), libigdfcl1 (>= 1.0.3627), ocl-icd-libopencl1
### Rhel8
Need to be added additional step with installing http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/ocl-icd-2.2.12-1.el8.x86_64.rpm
I'll also notify Driver team about such difference

**Review by p-durandin — APPROVED** [review]
OK for last changes
